bootstrap.js

```diff
@@ -200,12 +200,12 @@
             var getDefinition = function (name) {
                 return bundleDefinitions[name] =
                     bundleDefinitions[name] ||
-                        Promise.defer();
+                        Promise.resolve();
```

Comment:
These lines are not equivalent. Promise.defer() got replaced with `new Promise(…)` etc.